### PR TITLE
Dispatch `stamped` event as Juicy/juicy-html does

### PR DESCRIPTION
**Pull request by tomalec**
We need tests for this, but they should be added in `juicy-html` repo first https://github.com/Juicy/juicy-html/issues/33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,7 @@ To provide a state before element is upgraded, please use attributes.
 
 Name       | Detail                                     | Description
 ---        | ---                                        | ---
+`stamped`  | *Array* of *Node* s array of stamped nodes | Trigger every time content is (re-)stamped
```
Comment (warpech) [inline README.md]: Shouldnt' detail be ` *Array* of *Node* s` like in `juicy-html`?

Comment (tomalec) [reply README.md]: I don't get it. It is "Array of Nodes".

Comment (warpech) [reply README.md]: I can see some redundant text in that column. See the highlight on the below screenshot:

Comment (tomalec) [reply README.md]: ah! So you want to remove "array of stamped nodes"?

Comment (warpech) [reply README.md]: Yes, or move the description of the detail to the description column

Comment (tomalec) [reply README.md]: I gave it a second thought, I would say we need two descriptions: one to describe when the event is fired, another to describe the detail contents.

Comment (warpech) [reply README.md]: https://github.com/Juicy/imported-template/pull/32/commits/80eaeec96d8c2ca8df9f5f0f5489285883264a83 looks good to me

**Comment by warpech** [comment]
Setting as “Finished” because it is:
- released (with release notes) https://github.com/Juicy/imported-template/releases/tag/2.0.1
- included in a Finished release of SCF https://github.com/Starcounter/StarcounterClientFiles/releases/tag/1.0.0

**Pull request merged**